spark/v3.3/spark/src/main/java/org/apache/iceberg/spark/SparkReadOptions.java

```diff
@@ -32,6 +32,12 @@ private SparkReadOptions() {}
   // End snapshot ID used in incremental scans (inclusive)
   public static final String END_SNAPSHOT_ID = "end-snapshot-id";
 
+  // Start timestamp used in multi-snapshot scans (exclusive)
+  public static final String START_TIMESTAMP = "start-timestamp";
+
+  // End timestamp used in multi-snapshot scans (inclusive)
+  public static final String END_TIMESTAMP = "end-timestamp";
```

Comment:
The other option here is to reuse `as-of-timestamp` as the end timestamp, but I prefer the symmetric options(start-timestamp, end-timestamp). The UX would be better this way, I assume.